libs/libc/sched/CMakeLists.txt

```diff
@@ -26,11 +26,11 @@ set(SRCS
     clock_timespec_add.c
     clock_timespec_subtract.c
     clock_getcpuclockid.c
-    clock_getres.c)
-
-if(NOT CONFIG_CANCELLATION_POINTS)
```

Comment:
why remove the check of CONFIG_CANCELLATION_POINTS

Reply:
The way it worked previously:
- There were two versions of task_setcanceltype.c and task_testcancel.c, one in sched/task and one in libc/sched
- If cancelpoints=y then the sched/task versions are compiled, if cancelpoints=n then the libc versions are compiled
- The libc versions were just dummies that do nothing, besides provide the task_setcanceltype() and task_testcancel() symbols for compilation
- Now there is only 1 version, in libc, which means the libc version must always be compiled, otherwise the symbols are missing and the code will not compile